packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,116 +2,174 @@
-        border-color: GrayText;
+    .iui-slider-tick {
+      flex-direction: column;
+      inline-size: var(--_iui-slider-tick-thickness);
+
+      &::before {
+        inline-size: var(--_iui-slider-tick-thickness);
+        block-size: var(--_iui-slider-tick-size);
       }
     }
 
-    .iui-slider-track {
-      background-color: var(--iui-color-border);
-      @media (forced-colors: active) {
-        background-color: GrayText;
+    .iui-slider {
+      block-size: calc(var(--_iui-slider-thumb-size) + 2px);
+      touch-action: pan-x;
+
+      &::before {
+        grid-column-start: 1;
+        inline-size: 100%;
       }
     }
 
+    .iui-slider::before,
+    .iui-slider-track {
+      block-size: var(--_iui-slider-track-thickness);
+    }
+
+    .iui-slider-track {
+      inset-inline-start: 0;
+      inset-block-start: 50%;
+      transform: translateY(-50%);
+    }
+
+    .iui-slider-thumb {
+      transform: translateX(-50%);
+    }
+  }
+
+  &[data-iui-orientation='vertical'] {
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+      'max'
+      'content'
+      'min';
+    justify-items: center;
+
     .iui-slider-min,
-    .iui-slider-max,
+    .iui-slider-max {
+      justify-content: center;
+    }
+
+    .iui-slider-min {
+      margin-block-start: var(--_iui-slider-container-gap);
+    }
+
+    .iui-slider-max {
+      margin-block-end: var(--_iui-slider-container-gap);
+    }
+
+    .iui-slider-ticks {
+      flex-direction: column-reverse;
+      transform: translateX(50%);
+    }
+
     .iui-slider-tick {
-      user-select: none;
-      color: var(--iui-color-text-disabled);
-      @media (forced-colors: active) {
-        color: GrayText;
+      block-size: var(--_iui-slider-tick-thickness);
+
+      &::before {
+        inline-size: var(--_iui-slider-tick-size);
+        block-size: var(--_iui-slider-tick-thickness);
+      }
+    }
+
+    .iui-slider {
+      inline-size: calc(var(--_iui-slider-thumb-size) + 2px);
+      touch-action: pan-y;
+
+      &::before {
+        block-size: 100%;
+        grid-row: 1 / 3;
       }
     }
+
+    .iui-slider::before,
+    .iui-slider-track {
+      inline-size: var(--_iui-slider-track-thickness);
+    }
+
+    .iui-slider-track {
+      inset-block-end: 0;
+      inset-inline-start: 50%;
+      transform: translateX(-50%);
+    }
+
+    .iui-slider-thumb {
+      transform: translateY(50%);
+    }
+  }
+
+  &[data-iui-disabled='true'] {
+    cursor: not-allowed;
+    color: var(--iui-color-text-disabled);
+
+    --_iui-slider-rail-color: var(--iui-color-background-disabled);
+    --_iui-slider-track-color: var(--iui-color-border);
+    @media (forced-colors: active) {
+      --_iui-slider-track-color: GrayText;
+    }
+
+    .iui-slider {
+      pointer-events: none;
+    }
   }
 }
 
 .iui-slider-min,
 .iui-slider-max {
   user-select: all;
   display: flex;
-  justify-content: center;
 
   svg {
     @include mixins.iui-icon-style('m');
```

Comment:
can we use the `iui-svg-icon` class instead of duplicating the icon styles here?

Reply:
Yes, I've made this change.